applications/DEMApplication/custom_strategies/strategies/explicit_solver_strategy.cpp

```diff
@@ -1779,23 +1779,6 @@ namespace Kratos {
         r_model_part.GetCommunicator().SynchronizeVariable(PARTICLE_MOMENT);
     }
 
-    void ExplicitSolverStrategy::CleanEnergies() {
-        
-        KRATOS_TRY
-
-        ProcessInfo& r_process_info = GetModelPart().GetProcessInfo();
-        double& total_elastic_energy = r_process_info[PARTICLE_ELASTIC_ENERGY];
-        total_elastic_energy = 0.0;
-        double& total_inelastic_frictional_energy = r_process_info[PARTICLE_INELASTIC_FRICTIONAL_ENERGY];
-        total_inelastic_frictional_energy = 0.0;
-        double& total_inelastic_viscodamping_energy = r_process_info[PARTICLE_INELASTIC_VISCODAMPING_ENERGY];
```

Comment:
@maceligueta If we want to avoid the processInfo, Which could be the best option in your opinion? 

I was discussing this issue with @rlrangel and I was thinking about adding a pointer to a class storing these values in the strategy itself...